topics/cdnjs/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-created_by: Ryan Kirkman, Thomas Davis, Peter Dave, Matt Cowley
+created_by: Matt Cowley, Sven Sauleau, Tyler Caslin, Drew Freyling, Alex, Ryan Kirkman, Thomas Davis
 display_name: cdnjs
-github_url: https://github.com/cdnjs/cdnjs
+github_url: https://github.com/cdnjs/packages
 url: https://cdnjs.com/
 logo: cdnjs.png
 related: cdn, javascript, css, html, library, package, opensource, foss
 topic: cdnjs
 short_description: The \#1 free and open source CDN built to make life easier for developers.
 ---
-cdnjs, the #1 CDN that is open source and free to use. Fully supporting https, SPDY, http/2 and SRI on all library assets.
+cdnjs is a free and open source CDN service trusted by over 10% of websites, powered by Cloudflare. Making it faster and easier to load library files on your websites.
```

Comment:
Since the number is likely to change, it’d be great if we could omit it to reduce churn. What do you think about this:

```suggestion
A free and open source CDN service powered by Cloudflare. Making it faster and easier to load library files on your websites.
```